test/schema.test.ts

```diff
@@ -45,7 +45,7 @@ export function assertSchema(
             settings.required = true;
         }
 
-        const files = [resolve(BASE + group + "/main.ts")];
```

Comment:
Huh, why is this correct?

Reply:
@domoritz 
the 'resolve' function return the full path, 
So we were not building the 'userSymbols' correctly, when the path is the full path
Because when we check whether a file is the user's file we check it against a relative path
see: https://github.com/YousefED/typescript-json-schema/blob/master/typescript-json-schema.ts#L1603 

Please run the "test" script locally before my branch and run after my branch for reproduce